supabase/migrations/20230817110845_access_token.sql

```diff
@@ -0,0 +1,174 @@
+create table app.access_tokens(
+    id uuid primary key default gen_random_uuid(),
+    user_id uuid not null references auth.users(id),
```

Comment:
i think we should cascade deletes here since an access token shouldn't exist if a user is deleted
```suggestion
    user_id uuid not null references auth.users(id) on delete cascade,
```